src/js/lotto/lottoDomain.js

```diff
@@ -0,0 +1,42 @@
+import { $, WARNING_WHEN_NOT_IN_1000_UNITS } from "../shared/consts.js"
+import { removeCssToggleWhenResubmitting } from "./util/removeCssToggleWhenResubmitting.js";
+
+import lottoTicket from "./lottoTicket.js"
+
+const $lottoPurchaseInput = $('.lotto-purchase-input');
+const tagsShownAccordingToPurchaseStatus = [$('#purchased-lottos'), $('#lotto-winning-numbers-form')]
+
+const checkDisplayWhenResubmitting = () => {
+  tagsShownAccordingToPurchaseStatus.map(tag => { 
+    if (tag.style.display === "block") tag.style.display = 'none'
+  })
+}
+
+const passValidatitonLottoPurchaseAmount = amount => {
+  amount = Number(amount)
+
+  if (amount < 1000 || amount >= 1000000) {
+    checkDisplayWhenResubmitting()
+    return false
+  }
+
+  if (amount % 1000 !== 0) {
+    $lottoPurchaseInput.value = ""
+
+    alert(WARNING_WHEN_NOT_IN_1000_UNITS)
+    checkDisplayWhenResubmitting()
+    return false
+  }
+
+  return true
+}
+
+const lottoDomain = amount => {
+  if (passValidatitonLottoPurchaseAmount(amount)) {
+    tagsShownAccordingToPurchaseStatus.map(tag => tag.style.display = "block")
+    lottoTicket(amount)
+    removeCssToggleWhenResubmitting()
+  }
+}
```

Comment:
```suggestion
const lottoDomain = amount => {
  if (!passValidatitonLottoPurchaseAmount(amount)) return;
  tagsShownAccordingToPurchaseStatus.map(tag => tag.style.display = "block")
  lottoTicket(amount)
  removeCssToggleWhenResubmitting()
}
```
이렇게 미리 return을 해주면 더 좋답니다!
if 내부에 코드가 없을 수록 가독성이 좋아져요!

Reply:
아 책으로 공부한 내용인데 ㅠㅠ 더 주의 깊게 코드를 짜도록 하겠습니다.